spec/pluralrules.html

```diff
@@ -53,6 +53,82 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-GetOperands" aoid="GetOperands">
+       <h1>GetOperands (x)</h1>
+       <p>
+         When the GetOperands abstract operation is called with argument _x_ (which must be a finite Number value or a String representation of it), it returns an Object value with the following list of plural operands used by plural rules functions.
+       </p>
+
+       <emu-table id="table-plural-operands">
+         <emu-caption>Plural Rules Operands</emu-caption>
+         <table class="real-table">
+           <thead>
+             <tr>
+               <th>Symbol</th>
+               <th>Value</th>
+             </tr>
+           </thead>
+           <tr>
+             <td>n</td>
+             <td>absolute value of the source number (integer and decimals)</td>
+           </tr>
+           <tr>
+             <td>i</td>
+             <td>integer digits of n.</td>
+           </tr>
+           <tr>
+             <td>v</td>
+             <td>number of visible fraction digits in n, <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>w</td>
+             <td>number of visible fraction digits in n, <i>without</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>f</td>
+             <td>visible fractional digits in n, <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>t</td>
+             <td>visible fractional digits in n, <i>without</i> trailing zeros. </td>
+           </tr>
+         </table>
+       </emu-table>
+
+       <p>
+        The following steps are taken:
+       </p>
+
+       <emu-alg aoid="GetOperands">
+         1. Let _sv_ be ? ToString(_x_).
+         1. If _sv_ starts with *"-"*, then
```

Comment:
Unless I'm missing something, what you are proposing will not work.
If the user is passing _x_ as a String because he wants to pass `1.20` or `1.00` and I will cast it to _nv_ as a Number, and then do all computations based on the string balue of _nv_, I'm going to lose all the information that the user wanted to preserve by passing he string in the first place, no?